apisix/ssl.lua

```diff
@@ -252,9 +253,14 @@ function _M.check_ssl_conf(in_dp, conf)
         end
     end
 
-    local ok, err = validate(conf.cert, conf.key)
-    if not ok then
-        return nil, err
+    -- when cert or key is referenced by secret, avoid dynamic verification of it
```

Comment:
can we cache key and cert here with 
```
local new_ssl_value = secret.fetch_secrets(matched_ssl.value) or matched_ssl.value
```
And validate the `new_ssl_value `

Reply:
check like this: 
```
local ok, err = validate(new_ssl_value.cert, new_ssl_value.key )
```

Reply:
new_ssl_value as a whole is cacheable

Reply:
At present, only the MVP version is implemented, and a perfect caching mechanism will be considered in the future to reduce the time-consuming of a single request